.github/workflows/lcm_dreamshaper_cpp.yml

```diff
@@ -24,7 +24,7 @@ env:
 
 jobs:
   lcm_dreamshaper_v7_cpp-linux:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-22.04-8-cores
```

Comment:
Why did the worflows change?

Reply:
more source files => small runners failed to compile / link (presumably, because of memory consumptions)